_maps/map_files/Mafia/mafia_ball.dmm

```diff
@@ -0,0 +1,48 @@
+"a" = (/turf/closed/indestructible/riveted,/area/mafia)
+"b" = (/turf/closed/indestructible/riveted/plastinum,/area/mafia)
+"c" = (/turf/closed/wall/rust,/area/mafia)
+"d" = (/turf/open/floor/plating,/area/mafia)
+"e" = (/turf/open/floor/plating{icon_state = "platingdmg3"},/area/mafia)
+"f" = (/turf/open/floor/plating{icon_state = "panelscorched"},/area/mafia)
+"g" = (/obj/mafia_game_board,/turf/open/floor/plating{icon_state = "platingdmg3"},/area/mafia)
+"h" = (/obj/structure/grille/indestructable,/turf/open/floor/plating,/area/mafia)
+"i" = (/turf/open/floor/plating{icon_state = "platingdmg1"},/area/mafia)
+"j" = (/obj/machinery/door/airlock/maintenance_hatch{max_integrity = 99999; name = "Maintenance"},/obj/effect/mapping_helpers/airlock/locked,/turf/open/floor/plating,/area/mafia)
```

Comment:
use INFINITY instead of 99999, just in case

Reply:
good catch, thank you. i already caught one case of this earlier

Reply:
Map isn't map merged.